### DISCUSSION TITLE
Password stopped working on the SO terminal

### DISCUSSION THREAD

**Discussion by rejlive** [Security-Onion-Solutions/securityonion#8987] [Unsupported Versions] [answered]
Hi
The Security Onion (SO) VM user password for the terminal on the CentosOS 7 64 was no longer recognized by the the terminal. The password worked fine up until some unknown point and I do not know what point that is since I did not have to log in again until the entire installation and onion page was accessible via Windows 10 using IP address an email and separate password. When I logged out of SO VM and tried to log in again I noticed the password didnt work anymore. I also noticed that even though the password fails I can still log into the SO page from Windows 10 VM using the email and password the installation had me set up. 
Thanks for any help

**Comment by xfaith** [comment]
They are two separate accounts. One is for the web login, and one would be for the shell.
You should have two accounts for Security Onion
1.  Web login (email address + password)  -- use at https://the_ip_of_the_vm
3. Shell login (username/password) - use at the shell login from the VM

**Reply by rejlive** [reply]
Yes the shell login password does not work.

**Reply by dougburks** [reply]
Depending on how you configured the installation, you might be able to do one of the following:

- login to another shell account and then use sudo/root privileges to change the password on your non-working account

- boot the installation into single user mode and change the user password 

- boot from a LiveCD, mount the filesystem, and then change the user password

**Reply by rejlive** [reply]
Thanks Ill try single user mode. First time for me using SO. How do I do that?

I do not recall all the details of the configuration. I only setup 1 NIC even though it warned me to have 2 I chose suricata. Chose an email and password for the IP login from a Windows VM on the same network which still works despite not being logged into the shell. Standalone install, DHCP management interface, direct connect to internet, ETOPEN since it is free, IP for web interface, Basic install for recommended settings, 1 for suricata processes, and IP range.

**Reply by dougburks** [reply]
Depending on how you configured the installation, your base OS may be either CentOS or Ubuntu. You can search the Internet for "single user" and whatever distribution name is shown in `/etc/os-release`. Since this is your first time, it may be quicker and easier to just perform a fresh installation of Security Onion and make sure that you record your username and password in a password manager.

**Comment by rejlive** [comment]
I also have another question regarding the NIC.
My VMs include Windows server, Windows10, Kali Linux, Metasploitable2 and Security Onion.
How do I setup 2 NIC in a VM Network on the same subnet?

**Reply by dougburks** [reply]
Since this is unrelated to the discussion title of `Password stopped working on the SO terminal`, please create a new discussion with an appropriate title.

**Reply by rejlive** [reply]
Sorry I figured the NIT part out and I did not configure Onion right anyway so the password issue should not be be an issue after I use the eval version of onion. We can close this discussion.
Thanks Doug for the info on changing passwords

**Comment by rejlive** [comment, marked as answer]
Will reinstall using eval version
